controller/user-subscription.controller.js

```diff
@@ -0,0 +1,64 @@
+const { Sequelize } = require("sequelize");
+const database = require("../config/database.js");
+const sequelize = new Sequelize(database.development);
+const UserModel = require("../db/models/user.js");
+const User = UserModel(sequelize, Sequelize);
+const TelegramService = require("./telegram.service.js");
+
+module.exports = {
+  subscribeUser: async function (user) {
+    const [newUser, created] = await User.findOrCreate({
+      where: { userId: user.userId },
+      defaults: {
+        userId: user.userId,
+        userName: user.userName,
+      },
+      paranoid: false,
+    });
+    if (created) {
+      TelegramService.sendTextMessageToUser(
+        newUser,
+        "Subscription sucessful! You will receive the NASA's Astronomy Picture Of the Day automatically."
+      );
+    } else if (newUser.deletedAt) {
+      newUser.restore();
+      TelegramService.sendTextMessageToUser(
+        newUser,
+        "Subscription sucessful! You will receive the NASA's Astronomy Picture Of the Day automatically."
+      );
+    } else {
+      TelegramService.sendTextMessageToUser(
+        newUser,
+        "You are already subscribed! To unsubscribe use /unsubscribe"
+      );
+    }
+  },
+
+  unSubscribeUser: async function (user) {
+    //improve checkSubscribed
```

Comment:
se esse comentario for sobre algo que vc vai fazer depois, pode usar a tagzinha `TODO:` que é um padrão para situacoes assim:

`// TODO: improve checkSubscribed`